appeals/api/src/server/repositories/appeal.repository.js

```diff
@@ -278,6 +279,7 @@ const setAppealDecision = (id, { documentDate, documentGuid, version, outcome })
 					}
 				},
 				outcome,
+				// @ts-ignore
```

Comment:
The error this is ignoring is also present on main, so it wasn't introduced in this PR (not sure how it got past tscheck)